src/Database/Updater.php

```diff
@@ -148,13 +148,9 @@ public function getClassFromFile($file)
                 /*
                  * Class opening
                  */
-                if ($tokens[$i][0] === T_CLASS) {
-                    for ($j = $i + 1; $j < count($tokens); $j++) {
-                        if ($tokens[$j] === '{') {
-                            $class = $tokens[$i+2][1];
-                            break;
-                        }
-                    }
+                if ($tokens[$i][0] === T_CLASS && $tokens[$i-1][0] !== T_PAAMAYIM_NEKUDOTAYIM) {
```

Comment:
I'd rather keep the token constant comparison,  or switch the first comparison to a 'class' string literal altogether. But I will conform to either way.

Reply:
Hi, @LukeTowers. Sorry, couldn't answer earlier. 
I can't change the order of the token conditions, otherwise, it will fall to the second condition almost all the time. If I keep it as is, it will only evaluate the second condition when the first is `T_CLASS` which is just a couple of times for the test case at least.

For the `T_PAAMAYIM_NEKUDOTAYIM` is it ok if I just put a comment above specifying what kind of token it is. Something like 
`/* T_PAAMAYIM_NEKUDOTAYIM - token for '::' `

Reply:
I thought that it breaks the style of the parser. Right now it compares pre-tokenized values by their int value. It kind of makes sense when doing language parsers, but in our case its irrelevant I guess. So to recap, I will keep the order of the conditionals, but change the use of the constant to '::'. Did I get that right?

Reply:
Got it, should be okay now.